doc/api/worker_threads.md

```diff
@@ -1981,7 +1995,7 @@ const worker = new Worker(`
   `, { eval: true });
 
 worker.on('online', async () => {
-  const handle = await worker.startCpuProfile('demo');
+  const handle = await worker.startCpuProfile();
```

Comment:
Note that if this PR does not land before the original version of the API goes out in a release this will be a semver-major change since the API was was originally landed as experimental.

Reply:
You mean this two PRs should be merged into `main` branch before next version release ?

Reply:
Yeah, if the previous iteration happens to go out in the next release before this lands, the api changes made here would qualify it as a semver-major, so we want to avoid that.

Alternatively, we might want to mark this additional experimental just in case we need to make additional changes.

Reply:
The PR have been mereged. Thanks for your suggestions.